src/Stripe.net/Entities/SubscriptionsSchedules/SubscriptionSchedulePhasePlan.cs

```diff
@@ -0,0 +1,45 @@
+namespace Stripe
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    using Stripe.Infrastructure;
+
+    public class SubscriptionSchedulePhasePlan : StripeEntity
+    {
+        /// <summary>
+        /// Define thresholds at which an invoice will be sent, and the subscription advanced to a
+        /// new billing period.
+        /// </summary>
+        [JsonProperty("billing_thresholds")]
+        public SubscriptionSchedulePhasePlanBillingThresholds BillingThresholds { get; set; }
```

Comment:
Flagging that this is the same hash as the one on Subscription Item documented [here](https://stripe.com/docs/api/subscription_items/object#subscription_item_object-billing_thresholds). I did not want to re-use that class name as it's `SubscriptionItemBillingThresholds` and it's confusing.

@alexander-stripe suggested we unify though and rename both to `UsageBasedBillingThreshold` for example. Renaming would be a breaking change but since have a major revision shipping tomorrow or Friday maybe it's worth it? This also impacts the Options class(es).

Note that there is also `billing_thresholds` at the top level of this object and this one matches the one on Subscription and so I shared `SubscriptionBillingThresholds` because it felt like it made sense.

Reply:
I have clear fud here because openapi is generated based on client-metadata in pay-server. This is a separate discussion though.
Also whatever is in openapi can change without being considered a breaking change by anyone but a few of us.

I am fine using the `subscription_item_billing_thresholds` if you want, but I think it's for the wrong reasons. We should rename openapi in that case really.

Reply:
My point was that both resources use the same schema in OpenAPI, which means that in our backend code `subscription_item_billing_thresholds` is defined once and referenced in both resources. So if it ever changes, it would change for both resources at once and we'd need to reflect that in the bindings.

I agree the name is bad because it specifically mentions `subscription_item`, and should be changed.